src/Stratis.Features.Collateral/CheckCollateralCommitmentHeightRule.cs

```diff
@@ -36,6 +38,29 @@ public override Task RunAsync(RuleContext context)
                 PoAConsensusErrors.CollateralCommitmentHeightMissing.Throw();
             }
 
+            // Check that the commitment height is not less that of the prior block.
+            int release1324ActivationHeight = 0;
```

Comment:
Any chance we can test this with a unit test?

Reply:
Test added.